src/test/kotlin/no/nav/pdfgen/PdfGenITSpek.kt

```diff
@@ -113,6 +115,43 @@ object PdfGenITSpek : Spek({
         }
     }
 
+    describe("Using the image convert endpoint") {
+        it("Should render a document using input impage") {
+            val response = runBlocking<HttpResponse> {
+                client.post("http://localhost:$applicationPort/api/v1/genpdf/image/integration-test") {
+                    body = ByteArrayContent(testJpg, ContentType.Image.JPEG)
+                }
+            }
+            response.status.isSuccess() shouldEqual true
+            val bytes = runBlocking { response.readBytes() }
+            bytes shouldNotEqual null
+            Files.write(Paths.get("build", "jpg.pdf"), bytes)
+            // Load the document in pdfbox to ensure its valid
+            val document = PDDocument.load(bytes)
+            document shouldNotEqual null
+            document.pages.count shouldBeGreaterThan 0
+            println(document.documentInformation.title)
+            response.close()
```

Comment:
Should use `.use {}` on the HttpResponse object which is equivalent to Java's try-with-resources